core/java/com/google/instrumentation/stats/RpcConstants.java

```diff
@@ -13,20 +13,21 @@
 
 package com.google.instrumentation.stats;
 
+import com.google.instrumentation.common.Duration;
 import com.google.instrumentation.stats.MeasurementDescriptor.BasicUnit;
 import com.google.instrumentation.stats.MeasurementDescriptor.MeasurementUnit;
 import com.google.instrumentation.stats.ViewDescriptor.DistributionViewDescriptor;
+import com.google.instrumentation.stats.ViewDescriptor.IntervalViewDescriptor;
 
 import java.util.Arrays;
+import java.util.Collections;
 import java.util.List;
 
 /**
  * Constants for collecting rpc stats.
  */
 public final class RpcConstants {
-  /**
-   * Rpc tag keys.
-   */
+  // Rpc tag keys.
   public static final TagKey RPC_STATUS = TagKey.create("OpStatus");
   public static final TagKey RPC_CLIENT_METHOD = TagKey.create("method");
```

Comment:
I think going with OP_STATUS and METHOD probably makes the most sense but is out of scope for this pr. I'll put together something separately.